pkg/e2e/harness_runner/harness_runner.go

```diff
@@ -70,6 +73,7 @@ var _ = ginkgo.Describe("Test harness", ginkgo.Ordered, ginkgo.ContinueOnFailure
 			//	today
 			ginkgo.By("Running harness pod")
 			stopCh := make(chan struct{})
+			fmt.Println("Will timeout in", timeoutInSeconds, " seconds")
```

Comment:
We should be able to remove this print statement, another print statement already prints out the timeout duration when waiting for the harness pod ([pkg/common/runner/runner.go#130](https://github.com/openshift/osde2e/blob/b6ae4ef29f843efd4080c2ebf15ceb728ddc1f97/pkg/common/runner/runner.go#L130)):

```
2023/08/01 13:10:59 runner.go:130: Waiting for endpoints of osde2e-runner runner Pod with a timeout of 21600 seconds...
```